terms/pairings_or_bilinear_map.md

```diff
@@ -0,0 +1,44 @@
+# Pairings or Bilinear Maps
+
+Read the full article on pairings or bilinear
+maps [here](https://alinush.github.io/2022/12/31/pairings-or-bilinear-maps.html).
+
+## Definition
+
+A function $e$ is considered a bilinear function or map if it satisfies the following conditions:
+
+- Let $\mathbb{G}_1$, $\mathbb{G}_2$, and $\mathbb{G}_T$ be cyclic groups of order $n$. The bilinear pairing is
+  represented as $e : \mathbb{G}_1 \times \mathbb{G}_2 \rightarrow \mathbb{G}_T$.
+
+    - **Bilinearity**: $e(aP, bQ) = e(P, Q)^{ab}$.
+
+    - **Non-degeneracy**: $e(P, Q) \neq 1$.
+
+    - **Computability**: $e(P, Q)$ can be computed efficiently.
+
+## Properties
+
+- $e(P^{a}, Q^{b}) = e(P, Q)^{ab}$
+
+- $e(P, Q^{a}) = e(P, Q)^{a}$
+
+- $e(P, Q \cdot R) = e(P, Q) \cdot e(P, R)$
```

Comment:
wikipedia says it's e(P, Q + R) = e(P, Q) + e(P, R) 
but Vitalik's article says it's e(P, Q + R) = e(P, Q) * e(P, R)
I personally think the later is more credible

Reply:
![Screenshot 2023-12-24 at 08 43 28](https://github.com/sota-zk-lab/documents/assets/33181397/ac59a6e7-4ec4-4edc-95dc-990fc2de206b)

Reply:
then it should be e(P, Q + R) = e(P, Q) * e(P, R) instead of e(P, Q * R) = e(P, Q) * e(P, R) in order to be "consistent"

Reply:
It should be 'consistent' with other parts of this file. If I were to change it to '(+),' it would break other stated sections

Reply:
According to bilinearlity property, we have 2 example, the former is mine, the latter is your example below, we gonna evaluate them at a = P, b = Q:
$e(a, b) = 2^{a\*b}$ at a = P, b = Q
$2^{P\*Q}$
$=2^{2\*P \* 3\*Q}$
$=2^{P\*Q\*2\*3}$
$=(2^{P\*Q})^{2\*3}$
equals to 
$(2^{P\*Q})^{2\*3}$

$e(a, b) = 2^{log_3(a) \* log_3(b)}$ at a = P, b = Q
$2^{log_3(P) \* log_3(Q)}$
$=2^{log_3(2\*P) \* log_3(3\*Q)}$
$=2^{(log_3(P)+log_3(2)) \* (log_3(Q) + log_3(3))}$
$=2^{log_3(P)\*log_3(Q) +log_3(2)\*log_3(Q) + log_3(3)\*log_3(P) + log_3(2)\*log_3(3)}$
not equals to
$(2^{log_3(P) \* log_3(Q)})^{2\*3}$
$=2^{log3(P) \* log3(Q) \* 2 \* 3}$

This is my check on your example, it doesn't look right to me, so please check it again.